client/services/FeedbackGeneratorService.js

```diff
@@ -415,6 +415,20 @@ tie.factory('FeedbackGeneratorService', [
             'Please keep your code within the existing predefined functions',
             '-- we cannot process code in the global scope.'
           ].join(' '));
+        } else if (prereqCheckFailure.hasWrongLanguage()) {
+          WRONG_LANGUAGE_ERRORS.python.forEach(function(error) {
+            if (error.errorName === prereqCheckFailure.getWrongLangKey()) {
+              error.feedbackParagraphs.forEach(function(paragraph) {
+                if (paragraph.type === 'text') {
```

Comment:
Can we declare constants somewhere like PARAGRAPH_TYPE_TEXT and PARAGRAPH_TYPE_CODE? The validation I mentioned earlier should check that the value in the relevant location is one of the allowed paragraph types.